itou/gps/grist.py

```diff
@@ -0,0 +1,65 @@
+import httpx
+from django.conf import settings
+from django.urls import reverse
+from django.utils import timezone
+from huey.contrib.djhuey import task
+
+from itou.users.enums import UserKind
+from itou.utils.urls import get_absolute_url
+
+
+@task(retries=3, retry_delay=10)
+def add_records(doc_id, table_id, records):
+    if settings.GRIST_API_KEY is None:
+        return
+    url = f"https://grist.numerique.gouv.fr/api/docs/{doc_id}/tables/{table_id}/records"
+    response = httpx.post(
+        url,
+        headers={"Authorization": "Bearer " + settings.GRIST_API_KEY},
+        json={"records": records},
+    )
+    return response.raise_for_status().json()
+
+
+def get_user_admin_url(user):
+    return get_absolute_url(reverse("admin:users_user_change", args=(user.pk,)))
+
+
+def get_user_kind_display(user):
+    if user.kind == UserKind.EMPLOYER:
+        return "employeur"
+    elif user.kind == UserKind.PRESCRIBER:
+        if user.prescribermembership_set.filter(organization__is_authorized=True).exists():
```

Comment:
```suggestion
        if user.prescribermembership_set.filter(is_active=True, organization__is_authorized=True).exists():
```
Ou directement: `user.is_prescriber_with_authorized_org` :sweat_smile: